config/config.go

```diff
@@ -876,6 +876,10 @@ type MempoolConfig struct {
 	// Including space needed by encoding (one varint per transaction).
 	// XXX: Unused due to https://github.com/tendermint/tendermint/issues/5796
 	MaxBatchBytes int `mapstructure:"max_batch_bytes"`
+	// Only broadcast txs to up to this many peers.
+	// If we are connected to more than this number of peers, only send txs to
+	// the first ExperimentalMaxUsedOutboundPeers of them. If one of those peers goes offline, activate another peer.
+	ExperimentalMaxUsedOutboundPeers int `mapstructure:"experimental_max_used_outbound_peers"`
```

Comment:
Can you also explain here what setting it to 0 means?

Reply:
Done.